datajoint/fetch.py

```diff
@@ -172,6 +172,12 @@ def __call__(self, *attrs, offset=None, limit=None, order_by=None, format=None,
                     list((to_dicts if as_dict else lambda x: x)(ret[self._expression.primary_key])) if is_key(attribute)
                     else ret[attribute] for attribute in attrs]
                 ret = return_values[0] if len(attrs) == 1 else return_values
+                
+                if isinstance(ret,(np.ndarray)) and len(ret) > 0:
+                    main_type = type(ret[0])
+                    print(main_type)
```

Comment:
remove debug code

Reply:
thank you. I have made sure these have been removed.